src/renderer/viz/expressions/scale.js

```diff
@@ -0,0 +1,48 @@
+import BaseExpression from './base';
+import { number, mul } from '../expressions';
+import { implicitCast, checkType } from './utils';
+
+/**
+ * Scale a width value to keep feature width constant in real space (meters).
+ * This will change the width in pixels at different zoom levels to enforce the previous condition.
+ *
+ * @param {Number} input - pixel width at zoom level 0
+ * @return {Number}
+ *
+ * @example <caption>Keep feature width in meters constant.</caption>
+ * const s = carto.expressions;
+ * const viz = new carto.Viz({
+ *   width: s.scale(0.1)
+ * });
+ *
+ * @example <caption>Keep feature width in meters constant. (String)</caption>
+ * const viz = new carto.Viz(`
+ *   width: s.scale(0.1)
+ * `);
+ *
+ * @memberof carto.expressions
+ * @name scale
+ * @function
+ * @api
+ */
+export default class Scale extends BaseExpression {
```

Comment:
Does it make sense to set a default scale, so if the user writes `* scale()` everything remains the same?

Reply:
Another option will be to change the signature of `scale` so that it could be used as: `width: sqrt($amount) * scale( 8)`, meaning `sqrt($amount)` pixels at zoom 8. But this breaks the original approach (implementation-wise I don't care, both are very easy to implement, this feature is clearly about user-friendliness, not about implementaiton). @makella what do you think?

Reply:
width: scale(sqrt($amount)) * 0.0001 BAD, UGLY
width: scale(sqrt($amount), 7)
width: sqrt($amount) * scale(7)
width: ramp(zoomranges(7,10,12),
[blablabla, blablabla, scale(sqrt($amount), 7)]